cms/migrations/0024_courseware_carousel_manual_contents.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.1.7 on 2019-05-23 08:02
+
+from django.db import migrations, models
+import wagtail.core.blocks
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("cms", "0023_text_image_section_action_url")]
+
+    operations = [
+        migrations.AddField(
+            model_name="coursesinprogrampage",
+            name="contents",
+            field=wagtail.core.fields.StreamField(
+                [
+                    (
+                        "item",
+                        wagtail.core.blocks.PageChooserBlock(
+                            required=False,
+                            target_model=["cms.CoursePage", "cms.ProgramPage"],
+                        ),
+                    )
+                ],
+                blank=True,
+                help_text="The courseware to display in this carousel",
+            ),
+        ),
+        migrations.AddField(
+            model_name="coursesinprogrampage",
+            name="override_contents",
+            field=models.BooleanField(
+                blank=True,
+                default=False,
+                help_text="Manually select contents below. Otherwise displays all courses related associated with the program.",
```

Comment:
Thanks but actually this text is confusing me, It seems me it will only override courses related associated with the program. I am thinking it will also show up programs too not just the courses in programs. plz correct me over there or update the help text accordingly.